### DISCUSSION TITLE
Phần chat nhóm 5 Tin học

### DISCUSSION THREAD

**Discussion by HuuNghia10A12PhuNhuan** [HoangBao04/10A12-Nhom05#7] [General]
Chat bất cứ thứ gì bạn muốn, hãy xem đây như group nhóm thứ 2 sau group nhóm bên zalo. Chú ý: no văng tục and no spam

**Comment by HuuNghia10A12PhuNhuan** [comment]
aloha, m.ng qua đây chat nha, đừng chat bên zalo.

**Comment by HuuNghia10A12PhuNhuan** [comment]
@HoangBao04 @ThuyDung07-A12 @Dangchauu @DuyHieu14-10A12 **_cố làm bài điiiii_**

**Comment by HuuNghia10A12PhuNhuan** [comment]
@HoangBao04 cái vế 2 câu 4 thì t nghĩ là nó giống phần nêu ý tưởng hoặc là nó giống phần mô tả thuật toán hoặc là nó là phần lai của cả hai

**Reply by HuuNghia10A12PhuNhuan** [reply]
T thì thiên về việc nó giống phần "ý tưởng"

**Comment by HuuNghia10A12PhuNhuan** [comment]
Nếu nó giống phần ý tưởng thì ôg có thể vô phần Projects coi bài của t để tham khảo cách làm

**Comment by HuuNghia10A12PhuNhuan** [comment]
Còn nếu nó giống phần mô tả thuật toán thì nó có 2 cách: "liệt kê bước" và "sơ đồ khối"
 + Liệt kê bước thì ông có thể kiếm trên mạng cho nhanh để bik cách trình bày
 + Sơ đồ khối thì lên mạng coi cách làm sơ đồ khối và các loại khối khác nhau để trình bày

**Comment by HuuNghia10A12PhuNhuan** [comment]
@ThuyDung07-A12 @Dangchauu _**Làm bài nhanh đi !!!!!**_

**Comment by HuuNghia10A12PhuNhuan** [comment]
@HoangBao04 Câu 1 không có nguồn, tự làm à?

**Reply by Dangchauu** [reply]
https://1drv.ms/w/s!ApVMrMZyvroIadiB1NTCXksWJho       
T làm câu 2 ở đây nè

**Comment by ThuyDung07-A12** [comment]
@HuuNghia10A12PhuNhuan chỉ làm câu 8 với

**Comment by HuuNghia10A12PhuNhuan** [comment]
câu 8 ý đầu: có 2 cách: ... và ...

**Comment by HuuNghia10A12PhuNhuan** [comment]
vế sau:

**Comment by HuuNghia10A12PhuNhuan** [comment]
-Ý tưởng:
  + Gọi các quả bóng lần lượt là a.1, a.2, a.3, a.4, a.5
  + đặt giá trị max = a.1
  + lần lượt cho i chạy từ 2 đến 5, so sánh giá trị a.i với giá trị max, nếu a.i > max thì max nhận giá trị mới là a.i
  + khi i=5 thì dừng lại và thông báo giá trị max

**Reply by HuuNghia10A12PhuNhuan** [reply]
dựa trên đây mà làm

**Reply by HuuNghia10A12PhuNhuan** [reply]
![](https://f49-zpg.zdn.vn/5501628388774272319/37e3782560a2a9fcf0b3.jpg)

**Comment by HuuNghia10A12PhuNhuan** [comment]
Bước 1: Nhập N và dãy a.1, a.2, ..., a.N
Bước 2: Max <-- a.1; i <-- 2
Bước 3: Nếu i > N thì đưa ra giá trị max và kết thúc
Bước 4: Nếu a.i > max thì max <-- a.i
Bước 5: i <-- i+1 rồi quay lại bước 3

**Comment by HuuNghia10A12PhuNhuan** [comment]
@ThuyDung07-A12 @Dangchauu @HoangBao04 @DuyHieu14-10A12 có ai có khả năng trình bày file word đẹp mắt ko?

**Reply by DuyHieu14-10A12** [reply]
hmmm cho t thử đi do câu 3 của t phần columm không đủ để ghi ra code

**Reply by HuuNghia10A12PhuNhuan** [reply]
> hmmm cho t thử đi do câu 3 của t phần columm không đủ để ghi ra code

OK, trang trí đẹp mắt, có màu sắc tí nha

**Reply by HuuNghia10A12PhuNhuan** [reply]
t sẽ CB trước, khi xong thì t gửi cho ôg rồi ôg trang trí lại cho đẹp nha, trang trí sao cho điểm cao tí, chứ t hơi dở phần này

**Reply by DuyHieu14-10A12** [reply]
okok lun

**Comment by HuuNghia10A12PhuNhuan** [comment]
bây ơi

**Comment by HuuNghia10A12PhuNhuan** [comment]
cần 3 câu thành ngữ

**Comment by HuuNghia10A12PhuNhuan** [comment]
nên tạm thời tuần 2 này mình tổng hợp các câu thành ngữ rồi chọn ra 3 câu nha!!!!

**Comment by HuuNghia10A12PhuNhuan** [comment]
ai có ý kiến gì thì nhắn đi nha

**Reply by ThuyDung07-A12** [reply]
hạn là chủ nhật đúng k á?

**Reply by HuuNghia10A12PhuNhuan** [reply]
ừ, đr

**Comment by ThuyDung07-A12** [comment]
Có công mài sắt, có ngày nên kim.
tui thấy câu này hay á mng

**Reply by HuuNghia10A12PhuNhuan** [reply]
cũng ổn, ghi câu đó vô trong phần project đi

**Reply by HuuNghia10A12PhuNhuan** [reply]
bấm edit note rồi gắn cái câu đó vô vị trí 1/

**Reply by ThuyDung07-A12** [reply]
ok

**Reply by HuuNghia10A12PhuNhuan** [reply]
cái column "in progress", chọn phần "liệt kê" nha

**Reply by HuuNghia10A12PhuNhuan** [reply]
cái note có ghi chữ "liệt kê" á

**Reply by HuuNghia10A12PhuNhuan** [reply]
còn cái bà mới ghi là dùng để biểu quyết chọn 3 câu nào

**Reply by ThuyDung07-A12** [reply]
là sao tui k hiểu lắm

**Reply by HuuNghia10A12PhuNhuan** [reply]
> là sao tui k hiểu lắm

t sửa lại luôn rồi nha, lần sau thì cứ ghi vô cột đó

**Reply by ThuyDung07-A12** [reply]
ok cảm ơn nha

**Comment by ThuyDung07-A12** [comment]
Con hơn cha là nhà có phúc.
Đói cho sạch, rách cho thơm.
hai câu này thấy sao

**Reply by DuyHieu14-10A12** [reply]
tui thấy ổn quá nì cậu

**Reply by ThuyDung07-A12** [reply]
cậu sáng tác thơ đi cậu

**Reply by DuyHieu14-10A12** [reply]
tớ sợ sáng tác hay quá các cậu lại khen tui ngại

**Reply by ThuyDung07-A12** [reply]
thôi cậu đừng tự luyến

**Reply by HuuNghia10A12PhuNhuan** [reply]
> tớ sợ sáng tác hay quá các cậu lại khen tui ngại

:)))

**Reply by DuyHieu14-10A12** [reply]
wow mấy cậu không tin tui hả nì

**Reply by HuuNghia10A12PhuNhuan** [reply]
> wow mấy cậu không tin tui hả nì

chứ seo

**Comment by DuyHieu14-10A12** [comment]
Nhiễu điều phủ lấy giá gương. Người trong một nước phải thương nhau cùng. 2 câu nì phù hợp cho mùa covid vừa qua quá nì mng

**Reply by HuuNghia10A12PhuNhuan** [reply]
ổn á

**Reply by ThuyDung07-A12** [reply]
@DuyHieu14-10A12 sao viết sai câu của người ta z

**Reply by ThuyDung07-A12** [reply]
câu của người ta là z mà
Nhiễu điều phủ lấy giá gương
Người trong một nước phải thương nhau cùng

**Reply by ThuyDung07-A12** [reply]
ủa sao nó bị mất chữ điều với cùng z

**Reply by HuuNghia10A12PhuNhuan** [reply]
> ủa sao nó bị mất chữ điều với cùng z

t thấy vẫn bình thường mà

**Reply by DuyHieu14-10A12** [reply]
> ủa sao nó bị mất chữ điều với cùng z

sao cậu bị mất giống mình z=)))

**Comment by HuuNghia10A12PhuNhuan** [comment]
ê mà tụi bây chọn câu gì thì nhớ giải thích + 1 ít thông tin về câu đó trong đây nha

**Reply by HuuNghia10A12PhuNhuan** [reply]
thông tin để sau lày còn biết trước mà làm file 📁

**Reply by DuyHieu14-10A12** [reply]
à ukieee hỉu hỉu

**Comment by Dangphamquynhchau** [comment]
Tui góp thêm cho mng lựa chọn nha
Lửa thử vàng, gian nan thử sức
Chớ thấy sóng cả mà ngã tay chèo
Một miếng khi đói bằng một gói khi no

**Reply by HuuNghia10A12PhuNhuan** [reply]
có thông tin gì nữa ko? (Vd như: giải thích, ý nghĩa, tác dụng, lật ngược vđ,...)

**Reply by Dangphamquynhchau** [reply]
T thấy 2 câu trên đều nói về sự nỗ lực, quyết tâm, kiên trì  vượt qua đại dịch. Câu cuối là sự sẻ chia đùm bọc cũng hợp mùa covid giống ý tưởng của DHieu á. Còn giải thích zì đó thì lên mạng kham khảo

**Comment by Dangphamquynhchau** [comment]
Mà phân công nhóm 3 ng viết bài, còn 2 người trang trí word có ổn ko mng(ý kiến riêng)

**Reply by HuuNghia10A12PhuNhuan** [reply]
3 người viết bài thì sẽ có 3 lối viết khác nhau, t chưa chắc nó sẽ ntn
trang trí word thì chỉ cần 1 người thôi, những người còn lại đưa ý kiến (chỉ trang trí word sau khi đã viết xong bài nha)

**Comment by HuuNghia10A12PhuNhuan** [comment]
mng ơi, giờ mình bắt đầu chọn 3 câu ca dao hay nhất đi

**Reply by HuuNghia10A12PhuNhuan** [reply]
khi chọn thì nêu lên 1 vài lí do vì sao chọn câu đó mà ko phải câu khác
P.S: 1 người có thể chọn tối đa 3 câu (mỗi câu cần nêu lí do), 3 câu nào có nhiều lượt lựa chọn nhất sẽ đc chọn nha

**Reply by Dangphamquynhchau** [reply]
Lửa thử vàng gian nan thử sức nha✔
Lí do: khuyên con người dám thử thách bản thân, ko bỏ cuộc, nhẫn nại, luôn hướng tới những giới hạn mới. Và đây cũng là đức tính mỗi bạn trẻ cần phải có để thành công trong cuộc sống

**Reply by HuuNghia10A12PhuNhuan** [reply]
quên báo, hạn chọn 3 câu ca dao, tục ngữ, thành ngữ là tối T4 nha!!!

**Reply by Dangphamquynhchau** [reply]
Oke ỗn nha

**Reply by HuuNghia10A12PhuNhuan** [reply]
t thích các câu sau:

1/ Chớ thấy sóng cả mà ngã tay chèo
- Lí do: Khuyên con người cần có tinh thần tự cường, ko sợ khó khăn, ko sợ vấp ngã. Đặc biệt, trong thời điểm dịch Covid-19 đang hoành hành như thế này, câu ấy còn có ý nghĩa hơn khi chúng ta cần phải có ý chí mạnh mẽ, kiên cường và ý thức để có thể vượt qua đại nạn này.

2/ Lửa thử vàng, gian nan thử sức
- Lí do: *giống của Châu. Bonus: ý nghĩa của câu: chỉ khi gặp và vượt qua những khó khăn, thử thách trong cuộc sống thì ta mới có thể trở nên kiên cường hơn và đạt được thành công.

3/   Nhiễu điều phủ lấy giá gương
 Người trong một nước phải thương nhau cùng
- Lí do: khuyên con người nên yêu thương và giúp đỡ nhau. Trong thời buổi Covid thì câu này lại càng ý nghĩa hơn. VD: Con người Việt Nam đã tạo ra "máy ATM gạo" để giúp đỡ những người nghèo, cho đi miễn phí những hộp cơm cho những người vô gia cư và những người nghèo, ...

**Reply by ThuyDung07-A12** [reply]
tao thấy 3 câu của Nghĩa ok á

**Comment by Dangphamquynhchau** [comment]
Châu: cách đặt khoảng cách giữa các dòng
https://thuthuatoffice.net/khoang-cach-giua-cac-doan-trong-word.html  (có hình minh hoạ) 
Cách căn thụt lề
https://www-thegioididong-com.cdn.ampproject.org/v/s/www.thegioididong.com/hoi-dap/amp/huong-dan-3-cach-thut-le-dau-dong-trong-word-don-gian-dung-1392210?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16465585259196&referrer=https%3A%2F%2Fwww.google.com&amp_tf=T%E1%BB%AB%20%251%24s&ampshare=https%3A%2F%2Fwww.thegioididong.com%2Fhoi-dap%2Fhuong-dan-3-cach-thut-le-dau-dong-trong-word-don-gian-dung-1392210  (có hình minh hoạ)

**Reply by HuuNghia10A12PhuNhuan** [reply]
mặc dù ko bắt buộc nhưng t mong tụi bây sẽ tóm tắt lại thay vì chỉ gửi link. Link thì để trong phần "lấy ý tưởng từ" hoặc "cre" á!!!!

**Reply by Dangphamquynhchau** [reply]
Oke nha

**Comment by ThuyDung07-A12** [comment]
1/ Có công mài sắt, có ngày nên kim.
Vì có ý nghĩa răn dạy, giáo dục chúng ta cần phải nỗ lực, kiên trì và cố gắng không ngừng để đạt được ước mơ đó.
2/  Đói cho sạch, rách cho thơm
Vì khuyên răn sâu sắc giúp con người gìn giữ nhân phẩm và đạo đức.
3/ Nhiễu điều phủ lấy giá gương
Người trong một nước phải thương nhau cùng
Vì câu ca dao muốn khuyên nhủ con người cần phải biết đoàn kết, thương yêu nhau. Tinh thần đoàn kết thương yêu nhau là truyền thống của dân tộc.

**Comment by DuyHieu14-10A12** [comment]
1. Bán anh em xa, mua láng giềng gần. Vì câu này khuyên răn người ta nên ăn ở có tình có nghĩa, vui vẻ với hàng xóm láng giềng kề bên. Bởi anh em họ hàng dù là thân tình, máu mủ nhưng ở xa thì nếu có việc khẩn cấp, nghiêm trọng không thể có mặt nhanh chóng để giúp đỡ bằng người ngoài nhưng ở gần mình.
2. Ăn cháo, đá bát. Vì đó là câu thành ngữ phê phán những kẻ vong ơn bạc nghĩa với những người đã từng giúp mình vượt qua khó khăn hoạn nạn. Nhưng khi người giúp mình gặp khó khăn những kẻ đó lại ngoảnh mặt đi, thậm chí còn hãm hại ân nhân của mình.Nó sẽ răn dạy cho ta phải biết ơn, phải biết "Uống nước nhớ nguồn", nếu vong ơn bội nghĩa sẽ bị trừng trị.
3.  Dạy con từ thuở còn thơ
Dạy vợ từ thuở bơ vơ mới về.
Vì nó cho chúng ta biết rằng phải dạy con từ khi con nhỏ, hướng con theo những điều đúng đắn và răn đe – khuyên nhủ con khi con làm sai và làm ảnh hưởng đến người khác. Việc dạy con từ thuở nhỏ giúp con nhận thức được việc mình làm và giúp bé ngộ ra được nhiều điều – lý lẽ sống đúng đắn. Nếu như từ nhỏ không dạy con và khuyên bảo con khi sai trái khi lớn bé sẽ luôn cho những điều sai đó là đúng và bắt đầu hành động sai trái nối tiếp nhau.

**Reply by HuuNghia10A12PhuNhuan** [reply]
ông ơi, chỉ chọn 3 câu trong 7 câu trong project tuần 2 thôi nha

**Reply by DuyHieu14-10A12** [reply]
à oke ông để tui chỉnh lại

**Comment by DuyHieu14-10A12** [comment]
1.Con hơn cha là nhà có phúc
Vì câu trên muốn nhắn nhủ và gửi gắm chúng ta thông điệp rằng chúng ta những thế hệ đi không những kế thừa, giữ gìn, bảo tồn những thành quả, những thành tựu, những giá trị tốt đẹp và kinh nghiệm lao động và sáng tạo của cha ông ta và còn phát huy, phát triển những thành quả đó hơn nữa để xây dựng đất nước ngày càng giàu mạnh, sánh đôi cùng các cường quốc trên thế giới.Và đặc biệt tụi mình là học sinh, là măng non, tương lai của đất nước học tập và rèn luyện để hơn các thế hệ đi trước và góp phần cho công cuộc phát triển đất nước.
2.Đói cho sạch, rách cho thơm
Vì mang ý nghĩa: dù đói rách, cùng khổ hay gặp khó khăn vất vả cũng phải biết gìn giữ nhân cách và phẩm chất tốt đẹp, sống ngay thẳng, trong sạch.Và Giá trị của câu tục ngữ là đạo lý giáo hóa, khuyên răn sâu sắc giúp con người gìn giữ nhân phẩm và đạo đức.
3.Lửa thử vàng, gian nan thử sức
Vì câu trên muốn nói rằng Khó khăn thử thách chỉ làm cho con người ta có thêm cơ hội chứng tỏ bản thân mình. Câu tục ngữ có ý khuyên chúng ta hãy kiên trì vượt qua thử thách trong cuộc sống.Và chúng ta là học sinh thì đôi lúc sẽ có những lúc bị điểm kém thì đó chính là khó khăn muốn thử thách chúng ta, vì vậy phải luôn đứng dậy sau mỗi thất bại và cố gắng phấn đấu hơn.

**Comment by Dangphamquynhchau** [comment]
https://docs.google.com/document/d/1lw9zEtZRV1B9QvtPLVXAdIyvj-eQhAHSs08l1cLCciI/edit?usp=drivesdk

Tui gửi file nè, có gì giúp tui gắn vào project nha. Cảm ơn nhìu💕
